src/web/components/ArticleContainer.tsx

```diff
@@ -53,8 +53,8 @@ const articleAdStyles = css`
             margin-right: -328px;
         }
 
-        ${from.wide}  {
```

Comment:
Is this a bad rebase?

Reply:
I have this same code on master. Looks okay